pom.xml

```diff
@@ -31,16 +31,16 @@
         <module>stream-payment-order</module>
         <module>stream-portfolio</module>
         <module>stream-contacts</module>
-        <module>stream-compositions</module>
         <module>stream-loans</module>
         <module>stream-audiences</module>
+        <module>stream-compositions</module>
     </modules>
 
     <properties>
         <java.version>17</java.version>
-        <ssdk.version>15.2.4</ssdk.version>
-        <backbase-bom.version>2023.09.17-LTS</backbase-bom.version>
-        <boat-maven-plugin.version>0.16.6</boat-maven-plugin.version>
+        <ssdk.version>16.0.1</ssdk.version>
+        <backbase-bom.version>2023.12</backbase-bom.version>
```

Comment:
@modak-akshay It looks like you merged in the main branch into the support branch? Is that what you wanted to do? Then maybe better to just PR this to the main branch and upgrade your streams?

Reply:
Nevermind, I accidentally pushed that when I tried to create the support branch. This was before I realized you created it already. I fixed it now.